browser/suites.js

```diff
@@ -114,7 +114,17 @@ export function runSuites(reporter, childSuites, done) {
 function _runMocha(reporter, done, waited) {
   if (config.get('waitForFrameworks') && !waited) {
     var waitFor = (config.get('waitFor') || util.whenFrameworksReady).bind(window);
-    waitFor(_runMocha.bind(null, reporter, done, true));
+    waitFor(function() {
+      // NOTE: In Chrome57 test-fixture elements in the document might not get upgraded when
+      // there is a high GC https://bugs.chromium.org/p/chromium/issues/detail?id=701601
+      // We clone and replace the ones that weren't upgraded.
+      var tfs = document.querySelectorAll('test-fixture');
+      for (var i = 0, l = tfs.length, tf; i < l && (tf = tfs[i]); i++) {
```

Comment:
Let's go with simpler and more legible for loops, even if it adds a few more expressions to the loop body. There's just so many subtle ways to mistype this sort of code and have it almost work